### DISCUSSION TITLE
Does Penpot have shareable iframe embeds?

### DISCUSSION THREAD

**Discussion by jsmorabito** [uxbox/uxbox#1085] [Q&A]
Something like https://www.figma.com/blog/introducing-figmas-live-embed-kit/ ? Would be very cool to be able to embed specific artboards as well. For my use case, I'd like to embed designs in a shared knowledge base like Roam or Athens for a team to view and also to embed designs on my personal portfolio website 😊

*I searched the docs for "embed" and "iframe" which did not return any results

**Comment by myfunnyandy** [comment]
I'm afraid Penpot does not have embed optios yet, but is something that we definitely want to do eventually. Thanks for raising the topic, this kind of comments help us to understand the level of interest and use cases ;)

**Comment by Jamby93** [comment]
Since this is the first github result if searching for "penpot iframe" on google, I'll share our solution: by enabling the un-documented (but still present, apparently "only for development" https://github.com/penpot/penpot/blob/804146ae9a518e6b73ab28c2968ea9cf4b3af15f/common/src/app/common/flags.cljc#L70 ) flag "cors" (via PENPOT_FLAGS: enable-cors), we were able to embed iframes in our Confluence deploy. Without the flag enabled, the auth-token Cookie has SameSite Lax, which prevents it to be set if the top-level domain doesn't match the Penpot instance one (which is the case in any embedding, of course). With the flag enabled, SameSite is set to None, and the iframe works perfectly. 
This could of course raises some security concern (every origin is allowed to embed without any check), but since our Penpot and Confluence instances are segregated in our internal network, that is not a concern at all for us.

**Comment by SaadBazaz** [comment]
Much needed. It's pretty incomplete if there isn't the ability to add a Penpot frame (a direct one, instead of the entire file) in an Outline / Confluence / Notion doc.
